pagopa/io-app
# fix(IDPay): [IODPAY-115] New placeholder in IBAN onboarding page

## Short description
added new placeholder in IBAN onboarding page

<img width="250" src="https://user-images.githubusercontent.com/60693085/227950038-c648d547-66b4-4bc0-92b9-5c54f54fcd7d.jpeg">


## List of changes proposed in this pull request
- required svg added
- new SVG component used in IBAN onboarding page

## How to test
navigate to the page in question and make sure the new placeholder is correctly rendered
